app/questionnaire/questionnaire_manager.py

```diff
@@ -341,7 +325,15 @@ def _conditional_display(self, item):
 
         if item.schema_item:
 
-            item.skipped = ConditionalDisplay.is_skipped(item.schema_item, self)
+            item.skipped = False
+
+            if hasattr(item.schema_item, 'skip_condition') and item.schema_item.skip_condition:
+                rule = item.schema_item.skip_condition.__dict__
+                rule['when'] = rule['when'].__dict__
```

Comment:
It also no longer breaks for me. Worth investigating? or ignoring and putting it down to switching between branches etc?